### PR TITLE
Issue#61

### LINKED ISSUE #61: Criar arquivo para configuração de proxy
adicionar script para configurar http e https proxy para http://proxy:8080 nos programas:
- yarn
- git
- terminal linux

o script deve desconfigurar também.

### PR DESCRIPTION
**Pull request by Pr3d4dor**
fixed #61

### CODE REVIEW

File: devops/proxies-apply [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+yarn config set proxy http://proxy:8080
```
Comment (paulopieczarka) [inline devops/proxies-apply]: echo "Applying http_proxy for Yarn..."
...echo "Applying https_proxy for Yarn..."

File: devops/proxies-remove [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+yarn config delete proxy
```
Comment (paulopieczarka) [inline devops/proxies-remove]: echo "Removing http_proxy for Yarn..."
...echo "Removing https_proxy for Yarn..."

Comment (Pr3d4dor) [reply devops/proxies-remove]: O comando do yarn já imprime inforamções no terminal:
````
Bine@Bine-PC MINGW64 ~
$ proxies-apply
Bine@Bine-PC MINGW64 ~
$ proxies-apply
yarn config v1.5.1
success Set "proxy" to "http://proxy:8080".
Done in 0.05s.
yarn config v1.5.1
success Set "https-proxy" to "http://proxy:8080".
Done in 0.05s.
Applying http_proxy for GIT...
Applying https_proxy for GIT...
Applying http_proxy for Linux terminal...
Applying https_proxy for Linux terminal...
All done, proxies applied.
````

Comment (paulopieczarka) [reply devops/proxies-remove]: Certo. 👍

**Pull request merged**